support/upvote-moonlightbot.md

```diff
@@ -1,18 +1,59 @@
 # Upvote MoonlightBot
 
-By upvoting MoonlightBot on bot listing sites, you help us reaching a wider audience and speed up the development of new features, as well as improve the support system.
+By upvoting MoonlightBot on bot listing sites, you help us reach a wider audience and speed up the development of new features, as well as improve the support system.
 
-{% hint style="info" %}
-If you are in the support server, you can stock up your votes and earn exclusive rewards!
-{% endhint %}
 
-## Sites you can vote in
+## Why should you vote? And how does it help us?
+
+1. If we helped you or your server, you should vote so that other people can discover MoonlightBot's amazing features!
+2. Knowing you're supporting our work makes us motivated to do better. With more motivation, we can quickly ship new features to the bot!
+3. Some people are full of hatred, and they like to attack small bot developers like us. But your votes make us more resilient and allow us to continue offering the service.
 
-[![Bots for Discord](https://discords.com/bots/api/bot/314110696071888896/widget)](https://discords.com/bots/bot/314110696071888896/vote)
-[![discordbots.co](https://api.discordbots.co/bot/314110696071888896/widget)](https://api.discordbots.co/bot/314110696071888896/)
+## Perks/Rewards for voting
 
-[![MoonlightBot](https://bots.ondiscord.xyz/bots/314110696071888896/embed?showGuilds=true)](https://bots.ondiscord.xyz/bots/314110696071888896)
+For your first ever vote or after a 30 consecutive days of voting, you will receive one free week of MoonlightBot Premium. The bot will DM you with a button to claim it.
+By having voted in the past 72 hours, your max durations will be doubled. Voting also extends your premium duration by 48 hours. Be sure to vote for MoonlightBot often!
 
 {% hint style="warning" %}
-This list is not exahustive, some sites do not provide widget. For a more complete list, use the `/vote` command
+Ensure you allow Direct Messages from a mutual server so MoonlightBot can message you, otherwise you won't be able to claim your free week of Premium!
 {% endhint %}
+
+## Voting on each site
+
+### Bots for Discord
+
+To vote follow these steps:
+1. [Click here to visit the site](https://discords.com/bots/bot/314110696071888896/vote)
+2. Click the button that says "Upvote"
+3. Authorize log in with your Discord account if requested, then click Upvote again
+You have now successfully voted for MoonlightBot!
+
+### discordbots.co
+
+1. [Click here to visit the page](https://discordbots.co/bot/314110696071888896)
+2. Click "Login with Discord" in the top right corner
+3. Click the blue button that says "Authorize" to sign in
+4. Click the blue button that says "Vote"
```

Comment:
```suggestion
4. Click the blue button that says "Vote" located under the Brief Description of the bot page
```

This site's UI isn't optimal and amasses a ton of buttons in the section. This addition helps the voter locate the correct button.